tools/docker-compose/api.Dockerfile

```diff
@@ -70,7 +70,7 @@ VOLUME [ "/go/pkg/mod" ]
 
 RUN mkdir -p /bhapi/collectors/azurehound /bhapi/collectors/sharphound /bhapi/work
 RUN go install github.com/go-delve/delve/cmd/dlv@v1.23.0
-RUN go install github.com/cosmtrek/air@v1.44.0
```

Comment:
The organization for this repo has changed
Navigating to https://github.com/cosmtrek/air redirects to https://github.com/air-verse/air